### ISSUE TITLE
[feature]  taxonomic classification and markergene extraction

### ISSUE THREAD

**Issue by uloeber** [bactopia/bactopia#103]
Hi, 
I would like to ask whether you might include progenomes-classifier for taxonomic classification and markergene extraction.
https://github.com/AlessioMilanese/progenomes_classifier
Bests,
Ulrike

**Comment by rpetit3** [comment]
This looks promising. 

I think first it will need to be submitted to Bioconda (happy to help), once that is done I can look into adding it as a Bactopia Tool. I've submitted an issue to inquire about a Bioconda release.

**Comment by rpetit3** [comment]
Hi @uloeber 

The next version of Bactopia will include [CheckM](https://github.com/Ecogenomics/CheckM). Which does something similar to ProGenomes (taxon slassify based on marker genes). Do you think this would accomplish what you were hoping for with ProGenomes?

Side-note: Once Progenomes2 is released I will still plan on adding it in.

Cheers,
Robert

**Comment by rpetit3** [comment]
Hi @uloeber,

Like to get your thoughts on the above comment.

Thanks!

**Comment by uloeber** [comment]
Hi @rpetit3 ,
using checkM sounds reasonable. Another interesting tool Seems to be mytaxascan, which is even computing significance proxies. For downstream processing, I now worked a lot with the tools implemented in the microbial gene atlas. https://help.microbial-genomes.org/assessing-quality 
BTW it's amazing, how quickly you enhanced the functionality of bactopia. I'm much looking Forward using it for upcoming projects.

**Comment by rpetit3** [comment]
Thanks for the follow up! Your suggestion for mytaxascan looks very useful. I'm going to see what will be needed and hopefully add it as a Bactopia Tool. 

Thanks for the complement, we had two really good reviewers for our manuscript that provided great feedback and suggests. Similarly, let me know if there is anything you might find useful to be added for your upcoming projects. I have a feeling our needs, as well as others, are mutual when it comes to microbial genomics. So, most suggestions are probably going to be useful for many.

I'm going to close this one for now, and reopen another issue for mytaxascan/MIGA
